### PR TITLE
Install scripts

### PR DESCRIPTION
**Pull request by djodjo3**
- Additional scripts can now be run:
   => At pre-install (after files copy)
   => At post-install (after PS setup)
   => At init (just before webserver launch)
  At each step, will check if scripts are available and will run them
- Adding some more logs during script run
-  Script will crash if any step fails 
- Replace mysqladmin drop DB with mysql drop if exists DB

### CODE REVIEW

File: base/config_files/docker_run.sh [hunk 1]
```diff
@@ -32,6 +43,7 @@ if [ ! -f ./config/settings.inc.php  ]; then
 	if [ $PS_INSTALL_AUTO = 1 ]; then
 		RET=1
 		while [ $RET -ne 0 ]; do
+                    echo "\n* Checking if $DB_SERVER is available..."
```
Comment (Quetzacoalt91) [inline base/config_files/docker_run.sh]:
```suggestion
                   echo "\n* Checking if $DB_SERVER is available..."
```

**Comment by Quetzacoalt91** [comment]
Looks like there is a small indentation issue in the script. Can you please double check?

When that's done, you may run the file `base/generate_tags.sh` to spread the changes in each subfolder.

**Comment by PierreRambaud** [comment]
> Looks like there is a small indentation issue in the script. Can you please double check?
> 
> When that's done, you may run the file `base/generate_tags.sh` to spread the changes in each subfolder.

Yes, the script contains tabs and spaces, even before his change, this is why I approved :sweat_smile:

**Comment by djodjo3** [comment]
- Fixed the indentation issue (tentatively, but let me know)
- Fixed a typo (DN_NAME instead of DB_NAME...)
- Ran the `base/generate_tags.sh` script, does not seem to have changed much - I _suppose_ it's to be expected as I only updated the `docker_run.sh` file.

**Comment by Quetzacoalt91** [comment]
Thank you @djodjo3

**Pull request merged**